dev/check_function_signatures.py

```diff
@@ -347,9 +347,9 @@ def compare_signatures(base_branch: str = "master") -> list[Error]:
                             line=param_error.lineno,
                             column=param_error.col_offset + 1,
                             lines=[
-                                "[Non-blocking]",
+                                "[Non-blocking | Ignore if not public API]",
                                 param_error.message,
-                                f"This breaks existing `{func_name}` calls.",
+                                f"This change will break existing `{func_name}` calls.",
                                 "If this is not intended, please fix it.",
```

Comment:
/ai should we use `would break` instead of `will break`?

Reply:
/ai Let me provide more context. This message is used when we detect a change in a PR that breaks existing function calls (e.g. adding a new required argument).